packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageBox.tests.js

```diff
@@ -46,6 +46,7 @@ QUnit.module('MessageBox', moduleConfig, () => {
             const expectedOptions = {
                 icon: 'send',
                 stylingMode: 'text',
+                disabled: true,
             };
             const sendButton = this.$sendButton.dxButton('instance');
```

Comment:
Can you refactor and use here the same approach that you use in your new tests?
```
const button = Button.getInstance(this.$sendButton);
```

Better way: Also we can move it to module level. And use everywhere in tests
```
this.sendButton = Button.getInstance(this.$sendButton);
```

Reply:
[332595f30abb7fcb0abcc63677fca5a99e430962]